### PR TITLE
feat(grafana,telegraf): Pull metric data from nsq via telegraf

### PR DESCRIPTION
**Pull request by jchauncey**
This PR updates several parts of the monitoring stack to support our move to NSQ. It is in progress for now since we are waiting for telegraf to merge my PR - https://github.com/influxdata/telegraf/pull/1369

### CODE REVIEW

**Comment by arschles** [comment]
@jchauncey looks like you're downloading the telegraf deb and then installing telegraf in the Dockerfile at `telegraf/rootfs/Dockerfile`, but you also have checked in the telegraf binary. From the description of this PR, sounds like you're gonna download the telegraf binary going forward (either from the official releases or our custom build from our own object storage), so can we get rid of the checked-in binary?

File: telegraf/manifests/deis-monitor-telegraf-daemon.yaml [hunk 1]
```diff
@@ -35,6 +35,12 @@ spec:
             value: "true"
           - name: "ENABLE_PROMETHEUS"
             value: "true"
+          - name: "NSQ_CONSUMER_SERVER"
```
Comment (arschles) [inline telegraf/manifests/deis-monitor-telegraf-daemon.yaml:38]: nit: worth making the prefixes for all these env vars `NSQD_`?

**Comment by arschles** [comment]
@jchauncey the json diffs are a bit hard to parse, but the changed files themselves look ok. my [nitpick](https://github.com/deis/monitor/pull/110#discussion_r67244923) is not blocking for this PR. Also, thanks for removing the telegraf binary.

Code LGTM

File: telegraf/manifests/deis-monitor-telegraf-daemon.yaml (hunk 1 above)
Comment (jchauncey) [inline telegraf/manifests/deis-monitor-telegraf-daemon.yaml:38]: I did it so it matches the plugin name. Which is nsq_consumer

**Comment by jchauncey** [comment]
Yeah the json diffs suck. easy way to test it is to build and push the image and check the graphs out
